app/events_blueprint/views.py

```diff
@@ -134,15 +135,16 @@ def edit_event(logged_in_user,eventid):
     return response
 
 #search by name
-@events.route('/api/v1/events/<string:location>', methods=['GET'])
+@events.route('/api/v1/events/location/', methods=['GET'])
 @token_required
```

Comment:
do you really require a token to search? I think anybody should be able to search regardless of whether they are logged in or not